### PR TITLE
Fix potential Null Reference Exception

### PR DESCRIPTION
**Pull request by carldebilly**

### CODE REVIEW

**Comment by kapetan** [comment]
Have you seen the exception happen? The `ObjectStringifier` is only meant to be used internally. If there is a null pointer exception it's probably bug in the code.

**Comment by carldebilly** [comment]
It is caused if the code is "linked". Could be using _.net native_, the Xamarin AOT compiler or simply a linker removed code not used.

I found this problem running on Android with linked libraries.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+csharp_style_var_when_type_is_apparent = false
+csharp_style_var_elsewhere = false
+dotnet_style_qualification_for_field = false
+dotnet_style_qualification_for_property = false
+csharp_prefer_braces = true
+
+[*.csproj]
+indent_style = space
+tab_width = 2
+indent_size = 2
```
Comment (bonesoul) [inline .editorconfig]: i think this do not belong here and a rogue change. even if you want to submit this change, it deserves it's own PR that should be reviewed by the author of the library.

once you remove this rogue part, i guess @kapetan will be happy to review your null reference exception change.

**Comment by kapetan** [comment]
Should be fixed by https://github.com/kapetan/dns/commit/75ed4d4efc77fa65e52bdeee0f3351adf133dbbc
